<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Graph</class>
 <widget class="QWidget" name="Graph">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>405</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>475</width>
    <height>405</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>475</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true">  background-color: rgb(239, 239, 239);
</string>
  </property>
  <widget class="QCustomPlot" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>25</y>
     <width>475</width>
     <height>320</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>475</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> background-color: rgb(169, 217, 215);
  border: 1px solid gray;
color: black;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>350</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
  background-color: rgb(198, 201, 203);
  border: 1px solid gray;
  border-radius: 5px;
color: black;
}
</string>
   </property>
   <property name="text">
    <string>X</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>350</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
  background-color: rgb(198, 201, 203);
  border: 1px solid gray;
  border-radius: 5px;
color: black;
}
</string>
   </property>
   <property name="text">
    <string>Y</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="build_graph">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>380</y>
     <width>475</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
  background-color: rgb(169, 217, 215);
  border: 1px solid gray;
color: black;
}

QPushButton:pressed {
    background-color: rgb(90, 186, 203);
}</string>
   </property>
   <property name="text">
    <string>Build graph</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="x_start">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>350</y>
     <width>85</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;</string>
   </property>
   <property name="minimum">
    <number>-1000000</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>-10</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="x_end">
   <property name="geometry">
    <rect>
     <x>135</x>
     <y>350</y>
     <width>85</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;</string>
   </property>
   <property name="minimum">
    <number>-100</number>
   </property>
   <property name="maximum">
    <number>1000000</number>
   </property>
   <property name="value">
    <number>10</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="y_start">
   <property name="geometry">
    <rect>
     <x>285</x>
     <y>350</y>
     <width>85</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;</string>
   </property>
   <property name="minimum">
    <number>-1000000</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>-10</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="y_end">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>350</y>
     <width>85</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;</string>
   </property>
   <property name="minimum">
    <number>-100</number>
   </property>
   <property name="maximum">
    <number>1000000</number>
   </property>
   <property name="value">
    <number>10</number>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
